woocommerce/woocommerce-android
# Activate: Remote site credentials experiement

This PR just removes the site credentials experiment code, since the A/B test has concluded with the original variant as the winner.

**To test:**
1. Start the app (in a logged out state)
2. Tap on the Enter the store address button
3. Enter a self-hosted site address (with Jetpack)
4. Notice the email login screen is displayed
5. Finish logging and verify it works as expected
